iceoryx_posh/include/iceoryx_posh/internal/popo/building_blocks/chunk_queue_types.hpp

```diff
@@ -31,14 +31,14 @@ enum class ChunkQueueError
 struct ChunkTuple
 {
     ChunkTuple() = default;
-    ChunkTuple(iox::relative_ptr<mepoo::ChunkManagement> f_chunk) noexcept
+    explicit ChunkTuple(iox::relative_ptr<mepoo::ChunkManagement> f_chunk) noexcept
         : m_segmentId(f_chunk.getId())
         , m_chunkOffset(f_chunk.getOffset())
     {
     }
 
-    RelativePointer::id_t m_segmentId{iox::RelativePointer::NULL_POINTER_ID};
-    RelativePointer::offset_t m_chunkOffset{iox::RelativePointer::NULL_POINTER_OFFSET};
+    RelativePointer::id_t m_segmentId;
```

Comment:
why is this changed? are we sure this is correctly initialized with the default c'tor?

Reply:
I think I removed it as QAC complained. Which would be OK if there is only the c'tor with relative ptr to ChunkManagement. But yes, there is also a default c'tor. Will check it again